### ISSUE TITLE
Overlay not showing amount of items

### ISSUE THREAD

**Issue by JacobSoren** [notablackbear/poe_qolV2#32]
I have half a quad tab filled with items, but my overlay still shows 0/0. I have tried almost every version of the tool. And checked my settings over and over, seems to be no fix. 
https://prnt.sc/uk0ol9

**Comment by EgrenB3S23** [comment]
I had this issue and managed to resolve it after some back n forth with notablackbear. THere's plenty of possible causes of course, but here's how we solved my particular issue:

Do you have any **Alt-Art race rewards in remove-only tabs**, by any chance? While they're not yet accessible from Heist league *within the game*, they still show up in the API pull for your Heist stash, and worse yet, they are listed first.

So this means, if you have your chaos recipe tab as your first tab in-game, i.e. all the way to the left/top, you would normally set `tab=0` in `setup.ini`. But if you have any race rewards still sitting in remove-only tabs (the kind that you can take out in any league but only show up after a month), that "tab=0" will point to one of those instead!

If you have 40 normal tabs and 30 race reward tabs, the race reward tabs will occupy index#0-29, and the normal tabs will occupy index#30-69), and you therefore need to add 30 to whatever number you have `tab=#` set to.

My Heist league in-game stash:
#0:     **Currency tab**
#1:     **Chaos recipe tab**
#2..52: other tabs

...without race rewards, `tab=1` would be correct. But i have 45 race reward tabs. So when fetching my Heist league stash with the API, you get:
#0:	hidden race reward tab
#1:	hidden race reward tab
...
#2-44:	hidden race reward tab
#45:	**Currency tab**
#46:	**Chaos recipe tab**
#47-97: other tabs

In other words, count the nr of race reward tabs you have, in my case 45, and add that number to the original number you would've used if you didnt have race reward tabs, in my case, 1. 45+1 = 46

(i used Acquisition to quickly get that number, but you can count them manually on the official Path of Exile website, just click your character portrait on the top left, then go to Inventory. It can be slow to load though.

so I needed to set`tab=46` in setup.ini.
To confirm, I entered the API request URL for "Egren's Heist league tab nr 46" into my browser:
https://www.pathofexile.com/character-window/get-stash-items?league=Heist&tabIndex=46&accountName=Egren

To use this URL, edit the league, tabIndex, and accountName to your values, then just hit enter!


tl;dr: move your chaos recipe tab all the way to the front (position 0), and set "tab=X" where X is the number of hidden remove-only race reward tabs you have laying in wait for the 1 month mark, because those are listed first in the API.

Attached images display my setup in working order.
![1 setup ini values](https://user-images.githubusercontent.com/71558976/93671124-a816cf80-faa0-11ea-82db-87b81bf3e41f.png)
![2  in game tabs](https://user-images.githubusercontent.com/71558976/93671125-a8af6600-faa0-11ea-853d-e5871a0da8b9.png)
![3  Acquisition](https://user-images.githubusercontent.com/71558976/93671127-a947fc80-faa0-11ea-9949-d2357ea76423.png)

**Comment by Rn3333** [comment]
@EgrenB3S23 Worked for me, thank you sir.

**Comment by JacobSoren** [comment]
You are amazing, thanks so much!
